aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -39,6 +39,7 @@ zeroize = "1"
 bytes = "1.1.0"
 http = "0.2.4"
 tower = { version = "0.4.8" }
+shlex = { version = "1.1.0" }
```

Comment:
I borrowed this shell parsing dependency from what `rusoto` was doing, but open to other ideas here.

https://docs.rs/rusoto_credential/latest/src/rusoto_credential/profile.rs.html#417-427

Reply:
Ended up refactoring this to take the Java SDK's approach of just passing the whole string through `sh -c` / `cmd.exe /C` in [14ad7a9](https://github.com/awslabs/smithy-rs/pull/1356/commits/14ad7a9084a5047d8863f6038e675663738cadd4)